hw/ip/otp_ctrl/data/otp_ctrl_sec_cm_testplan.hjson

```diff
@@ -27,271 +27,271 @@
+      tests: ["otp_ctrl_parallel_lc_esc", "otp_ctrl_sec_cm"]
     }
     {
       name: sec_cm_part_data_reg_integrity
       desc: "Verify the countermeasure(s) PART.DATA_REG.INTEGRITY."
       milestone: V2S
-      tests: []
+      tests: ["otp_ctrl_init_fail"]
     }
     {
       name: sec_cm_part_data_reg_bkgn_chk
       desc: "Verify the countermeasure(s) PART.DATA_REG.BKGN_CHK."
       milestone: V2S
-      tests: []
+      tests: ["otp_ctrl_check_fail"]
     }
     {
       name: sec_cm_part_mem_regren
       desc: "Verify the countermeasure(s) PART.MEM.REGREN."
       milestone: V2S
-      tests: []
+      tests: ["otp_ctrl_dai_lock"]
     }
     {
       name: sec_cm_part_mem_sw_unreadable
       desc: "Verify the countermeasure(s) PART.MEM.SW_UNREADABLE."
       milestone: V2S
-      tests: []
+      tests: ["otp_ctrl_dai_lock"]
     }
     {
       name: sec_cm_part_mem_sw_unwritable
       desc: "Verify the countermeasure(s) PART.MEM.SW_UNWRITABLE."
       milestone: V2S
-      tests: []
+      tests: ["otp_ctrl_dai_lock"]
     }
     {
       name: sec_cm_lc_part_mem_sw_noaccess
       desc: "Verify the countermeasure(s) LC_PART.MEM.SW_NOACCESS."
       milestone: V2S
-      tests: []
+      tests: ["otp_ctrl_dai_lock"]
     }
     {
       name: sec_cm_access_ctrl_mubi
       desc: "Verify the countermeasure(s) ACCESS.CTRL.MUBI."
       milestone: V2S
-      tests: []
+      tests: ["otp_ctrl_dai_lock"]
     }
     {
       name: sec_cm_token_valid_ctrl_mubi
       desc: "Verify the countermeasure(s) TOKEN_VALID.CTRL.MUBI."
       milestone: V2S
-      tests: []
+      tests: ["otp_ctrl_smoke"]
     }
     {
       name: sec_cm_lc_ctrl_intersig_mubi
       desc: "Verify the countermeasure(s) LC_CTRL.INTERSIG.MUBI."
       milestone: V2S
-      tests: []
+      tests: ["otp_ctrl_dai_lock"]
     }
     {
       name: sec_cm_test_bus_lc_gated
       desc: "Verify the countermeasure(s) TEST.BUS.LC_GATED."
       milestone: V2S
-      tests: []
+      tests: ["otp_ctrl_smoke"]
     }
     {
       name: sec_cm_direct_access_config_regwen
       desc: "Verify the countermeasure(s) DIRECT_ACCESS.CONFIG.REGWEN."
       milestone: V2S
-      tests: []
+      tests: ["otp_ctrl_regwen"]
     }
     {
       name: sec_cm_check_trigger_config_regwen
       desc: "Verify the countermeasure(s) CHECK_TRIGGER.CONFIG.REGWEN."
       milestone: V2S
-      tests: []
+      tests: ["otp_ctrl_smoke"]
     }
     {
       name: sec_cm_check_config_regwen
       desc: "Verify the countermeasure(s) CHECK.CONFIG.REGWEN."
       milestone: V2S
-      tests: []
+      tests: ["otp_ctrl_smoke"]
     }
     {
       name: sec_cm_macro_mem_integrity
       desc: "Verify the countermeasure(s) MACRO.MEM.INTEGRITY."
       milestone: V2S
-      tests: []
+      tests: ["otp_ctrl_macro_err"]
     }
     {
       name: sec_cm_macro_mem_cm
       desc: "Verify the countermeasure(s) MACRO.MEM.CM."
       milestone: V2S
-      tests: []
+      tests: ["N/A"]
```

Comment:
From hjson description, this might vary depends on each vendor.
https://github.com/lowRISC/opentitan/blob/master/hw/ip/otp_ctrl/data/otp_ctrl.hjson#L1063
I could put a generic otp_ctrl_macro_err test, but it really depends on how vendor's DV engineer plan to verify it.

Reply:
just out of curiosity, don't we have a test that ensures a macro err will propagate into the open source side? 
If we have that, couldn't we list that here? I think it covers the countermeasure from an open source perspective.

Reply:
I can add support in DVSim to ignore testplan entries for which tests cannot be developed.

Reply:
Hey@tjaychen, thanks for the comments. This is my understanding, please correct me if I am wrong -
I saw there are two entries on countermeasure:
1). "MACRO.MEM.INTEGRITY" https://github.com/lowRISC/opentitan/blob/master/hw/ip/otp_ctrl/data/otp_ctrl.hjson#L1057
  For this one, on open source and close source, we probe the ECC function to create a otp macro error. Then we check status and alerts.
2). "MACRO.MEM.CM"
https://github.com/lowRISC/opentitan/blob/master/hw/ip/otp_ctrl/data/otp_ctrl.hjson#L1063
  This is where I put `N/A` because I thought it was specifically for additional vendor-specific countermeasures that is different from ECC errors.
  So currently we do not have any open source test entries for that. I believe all the tests would be developed in close source.
What do you think?

Reply:
@sriyerg Thank you for offering that. Maybe let us wait to see what Tim think about this entry, then decide if this is needed :)

Reply:
> Hey @tjaychen, thanks for the comments. This is my understanding, please correct me if I am wrong - I saw there are two entries on countermeasure: 1). "MACRO.MEM.INTEGRITY" https://github.com/lowRISC/opentitan/blob/master/hw/ip/otp_ctrl/data/otp_ctrl.hjson#L1057 For this one, on open source and close source, we probe the ECC function to create a otp macro error. Then we check status and alerts. 2). "MACRO.MEM.CM" https://github.com/lowRISC/opentitan/blob/master/hw/ip/otp_ctrl/data/otp_ctrl.hjson#L1063 This is where I put `N/A` because I thought it was specifically for additional vendor-specific countermeasures that is different from ECC errors. So currently we do not have any open source test entries for that. I believe all the tests would be developed in close source. What do you think?

yes you're definitely right. Sorry I should have clarified my question. Is this error signaling using the same port as the macro.mem.integrity error? I had assumed it was a different port, but I think I misremembered.

Reply:
Sorry, a bit late to the party. As @cindychip mentioned, `MACRO.MEM.CM` was intended as a catch-all for additional countermeasures inside the closed source shim, excluding the ones already captured by standardized error codes known on the open-source side (such as ECC). 

These will likely use different alert signalling paths through `ast` / `sensor_ctrl` and we do not really have a good way of knowing what to test there on the open-source side.

One thing we could do is exercise the `otp_alert_src_o` path here that is tied off in the generic model:

https://github.com/lowRISC/opentitan/blob/c4e9f84094831bae578803be809a827667f28c80/hw/ip/otp_ctrl/rtl/otp_ctrl.sv#L782

But I think ideally this would be something that is being tested in the foundry database.

Reply:
Good point, I can add an assertion for `otp_alert_o`.